bin/restartServices

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+#
+# restartServices
+# Simple wrapper to restart Apache and PHP FPM services; this can be specified in sudoers file as specifically allowed.
+#
+
+set -e
+set -u
+
+service apache2 restart
```

Comment:
No because the whole point is that _this_ command, ie `/path/to/bin/restartServices`,  goes in `sudoers` so that it's the only thing the unprivileged user can elevate  themselves for.

Since this script is already being run as root, there's no need to use `sudo`. 

If sudoers contained `/sbin/service` then the unprivileged user could fully control any service. 

If might be worth doing an `id - u` check here to error gracefully if not run as root, or possibly to recall itself under `sudo`.